### ISSUE TITLE
[Feature] Support Python API

### ISSUE THREAD

**Issue by DjangoPeng** [cnosdb/cnosdb#802]
Since 2019, "AI for Systems and Systems for AI" has become a trend. 

As we all know, TensorFlow and PyTorch are the most popular AI frameworks nowadays. 

However, I have noticed that CnosDB does not yet support a Python API, which could potentially lead to a loss of users such as data scientists and data engineers. 

In contrast, InfluxDB, a competitor of CnosDB, released the first version of a Python Client for interacting with InfluxDB in 2013. (Reference: https://github.com/influxdata/influxdb-python). 

I believe it is time for CnosDB to support a Python API.

**Comment by roseboy-liu** [comment]
For now, we give users an HTTP RESTful API to access cnosdb. Here are some [examples](https://docs.cnosdb.com/guide/application/application.html), we really haven't given python examples yet. We also support cnosdb access via Arrow Flight SQL, but python doesn't have official support yet, but it's close. Related to #792

**Comment by roseboy-liu** [comment]
@Subsegment   We could provide a python connector using HTTP RESTful api.

**Comment by DjangoPeng** [comment]
LGTM.

What's the integration plan of python connector? A new repo or something else? @Subsegment @roseboy-liu

**Comment by cnoshb** [comment]
Plan to go production on 7th next month. repo will be available here: 
https://github.com/cnosdb/cnosdb-client-python 
@DjangoPeng

**Comment by DjangoPeng** [comment]
Nice move!

I'd like to contribute some use cases of AI :) .

**Comment by Subsegment** [comment]
CnosDB Python Connector DEMO have be preliminarily completed，please try it out and bring up some issues if you have better suggestions :) 
@DjangoPeng

**Comment by DjangoPeng** [comment]
👍 Gotcha.  @Subsegment

**Comment by cnoshb** [comment]
@Subsegment  Can this issue close?

**Comment by DjangoPeng** [comment]
Basically speaking, we have Python Client to connect CnosDB as mentioned above.

From this perspective, we have Python API now and this issue could be closed.

While the interface of Python Client is lack of `Create Table API`, we should open a specifically new issue to track it.

**Comment by DjangoPeng** [comment]
/cc @cnoshb @Subsegment @roseboy-liu 

Closed.
